### PR TITLE
8345767: javax/swing/JSplitPane/4164779/JSplitPaneKeyboardNavigationTest.java fails in ubuntu22.04

### PR DESCRIPTION
**Pull request by prsadhuk**
javax/swing/JSplitPane/4164779/JSplitPaneKeyboardNavigationTest.java fails in ubuntu22.04 citing

```
Verifier 1 failed, rightButton2 is not focus owner,F6 doesn't transfer focus to the right/bottom side of the splitpane
Verifier 2 passed
Verifier 3 passed
Verifier 4 passed
----------System.err:(11/842)----------
java.lang.RuntimeException: Test failed, verifiers 1, failed for com.sun.java.swing.plaf.motif.MotifLookAndFeel 
```

seems like focus is not transferred in time but is very intermittent...Added stability fixes with waitForIdle, delay and test passed in several OCI systems without issue

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8345767](https://bugs.openjdk.org/browse/JDK-8345767): javax/swing/JSplitPane/4164779/JSplitPaneKeyboardNavigationTest.java fails in ubuntu22.04 (**Bug** - P4)

### Reviewers
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - **Reviewer**)
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Committer) Review applies to [0b3de433](https://git.openjdk.org/jdk/pull/22631/files/0b3de4330105d1853203bd4a8ad000648c886a45)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/22631/head:pull/22631` \
`$ git checkout pull/22631`

Update a local copy of the PR: \
`$ git checkout pull/22631` \
`$ git pull https://git.openjdk.org/jdk.git pull/22631/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 22631`

View PR using the GUI difftool: \
`$ git pr show -t 22631`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/22631.diff">https://git.openjdk.org/jdk/pull/22631.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/22631#issuecomment-2526976134)
</details>

### CODE REVIEW

File: test/jdk/javax/swing/JSplitPane/4164779/JSplitPaneKeyboardNavigationTest.java
Comment (kumarabhi006) [inline test/jdk/javax/swing/JSplitPane/4164779/JSplitPaneKeyboardNavigationTest.java:1]: Few changes can be done.

1. Frame title can be added.
2. Calling frame.pack() and frame.setSize() is redundant.
3. Few lines are crossing the column limit. (Including comments)

**Review by honkar-jdk — APPROVED** [review]
Apart from the following suggestions, test changes LGTM and work as expected on x11 & wayland.

Ln#174: Is it better to move the button click to the center of the btn for stability?

```
 robot.mouseMove(buttonLoc.x + button.getWidth() / 2,
                 buttonLoc.y + button.getHeight() / 2);
```
Ln#230: `frame = null` is not required in disposeFrame() and can be removed.

**Review by DamonGuy — APPROVED** [review]
I only checked this on x11, but works as I expect on Ubuntu 22.04. Also ran it with 50 repeats and it looks OK, however for similar Ubuntu 22.04 fixes, I've had some 50 repeat jobs pass while the failure was still intermittent.

I believe this test was a part of the suite of Ubuntu 22.04 tests that were ProblemListed. Is there any plan to remove the test from it? Or leave it as is for now?

Otherwise, changes look good.

**Comment by honkar-jdk** [comment]
@prsadhuk  Ran test on CI multiple times (repeat count 20) and the test failed on 3rd run.
Looks like intermittent issue is still present.

**Comment by prsadhuk** [comment]
> Apart from the following suggestions, test changes LGTM and work as expected on x11 & wayland.
> 
> Ln#174: Is it better to move the button click to the center of the btn for stability?
> 
> ```
>  robot.mouseMove(buttonLoc.x + button.getWidth() / 2,
>                  buttonLoc.y + button.getHeight() / 2);
> ```
> 
> Ln#230: `frame = null` is not required in disposeFrame() and can be removed.

ok..good point...updated...

**Review by honkar-jdk — APPROVED** [review]
Updated changes LGTM

**Comment by DamonGuy** [comment]
If what @honkar-jdk said is true, then best to leave it on the ProblemList. But then isn't the issue not properly fixed? Is this worth integrating as is?

**Pull request closed without merging**